pom.xml

```diff
@@ -58,7 +58,7 @@
     <scm>
         <url>https://github.com/ballerina-platform/ballerina-distribution.git</url>
         <developerConnection>scm:git:https://github.com/ballerina-platform/ballerina-distribution.git</developerConnection>
-        <connection>scm:git:https://github.com/ballerina-platform/ballerina-distribution.git</connection>
+        <connection>scm:git:https://github.com/NipunaMadhushan/ballerina-distribution.git</connection>
```

Comment:
@NipunaMadhushan This has to be restored to `ballerina-platform`

Reply:
@keizer619 addressed the suggestions.